<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWindowProfessor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1221</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabProfessorProfile">
     <attribute name="title">
      <string>Profile</string>
     </attribute>
     <widget class="QLabel" name="label1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label1_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Full name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label1_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Gender:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label1_4">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>40</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label1_5">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>100</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Address:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label1_6">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>150</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Phone number:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label1_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Degree:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbID">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbFullName">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbGender">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditEmail">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>40</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditAddress">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>100</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPhone">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>150</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="lbDegree">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>180</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabProfessorSchedule">
     <attribute name="title">
      <string>Schedule</string>
     </attribute>
     <widget class="QTableView" name="tvProfessorSchedule">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>121</y>
        <width>1181</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabStudentGrade">
     <attribute name="title">
      <string>Student Grade</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>1191</width>
        <height>391</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QTableView" name="tvStudentGrade">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>1151</width>
         <height>351</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="labelCourseTabStGrade">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Course:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelStFullNameTabStGrade">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Student full name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelStGradeTabStGrade">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Grade:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUpdateGrade">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>70</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Update Grade</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="dsbStGrade">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>10</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="maximum">
       <double>4.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="labelCourseTabGrade">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelStFullNameTabGrade">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>70</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelSemesterTabGrade">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>20</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Semester:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>70</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Term:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbSemesterTabGrade">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>20</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lbTermTabGrade">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>70</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>115</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Student id:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbStudentIdTabGrade">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabChangePassword">
     <attribute name="title">
      <string>Update password</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>531</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QLabel" name="labelRepeatNewPasswd">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>190</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Repeat new password:</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelNewPasswd">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>130</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>New password:</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelOldPasswd">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Old password:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditOldPass">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>40</y>
         <width>271</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditNewPass">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>130</y>
         <width>271</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditNewPassRepeat">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>190</y>
         <width>271</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="labelError">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>240</y>
         <width>271</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelErrorOldPass">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>80</y>
         <width>271</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnUpdatePasswd">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>340</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Update password</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
